arch/riscv/chunkset_rvv.c

```diff
@@ -0,0 +1,95 @@
+/* chunkset_rvv.c - General version of chunkset
+ * Copyright (C) 2023 SiFive, Inc. All rights reserved.
+ * Contributed by Alex Chiang <alex.chiang@sifive.com>
+ * For conditions of distribution and use, see copyright notice in zlib.h
+ */
+
+#include "zbuild.h"
+
+/*
+ * It's not a optimized implemantation using RISC-V RVV, but a general optimized one.
+ *
+ * RISC-V glibc would enable RVV optimized memcpy at runtime by IFUNC,
+ * so we prefer using large size chunk and copy memory as much as possible.
+ */
+#define CHUNK_SIZE 32
+
+/* We don't have a 32-byte datatype for RISC-V arch. */
+typedef struct chunk_s {
+    uint8_t data[CHUNK_SIZE];
+} chunk_t;
+
+static inline void loadchunk(uint8_t const *s, chunk_t *chunk) {
+    memcpy(chunk->data, s, CHUNK_SIZE);
```

Comment:
The original casts s to `uint8_t` because memcpy does not expect `const`. Perhaps add that back to avoid some compiler warnings?
`memcpy(chunk, (uint8_t *)s, sizeof(uint64_t));`

Reply:
Thanks, it's done.